internal/controller/policies/policies.go

```diff
@@ -98,3 +98,13 @@ func NewSecretPostIssuancePolicyChain(fieldManager string) Chain {
 		SecretTemplateMismatchesSecretManagedFields(fieldManager),
 	}
 }
+
+// NewTemporaryCertificatePolicyChain includes policy checks for ensuing a
+// temporary certificate is valid.
+func NewTemporaryCertificatePolicyChain() Chain {
```

Comment:
Good to see these in one place 👍🏼 I didn't even know there was a policy chain for the temporary cert check